testsuite/tests/libratest/smoke_test.rs

```diff
@@ -874,3 +874,28 @@ fn test_client_waypoints() {
         .test_validator_connection()
         .is_err());
 }
+
+#[test]
+fn test_malformed_script() {
+    let (_swarm, mut client_proxy) = setup_swarm_and_client_proxy(1, 0);
+    client_proxy.create_next_account(false).unwrap();
+    client_proxy
+        .mint_coins(&["mintb", "0", "100"], true)
```

Comment:
Interesting, 0x0 was reserved. 

nit, mind adding a note what it test trying to catch? Like minting against reserved address.

Reply:
LGTM otherwise

Reply:
We aren't minting to 0x0 but rather create an account in client with local index 0.